src/algebra/group/basic.lean

```diff
@@ -648,3 +649,107 @@ lemma commutator_element_def  {G : Type*} [group G] (g₁ g₂ : G) :
   ⁅g₁, g₂⁆ = g₁ * g₂ * g₁⁻¹ * g₂⁻¹ := rfl
 
 end commutator
+
+/-! ### Order dual -/
+
+open order_dual
+
+@[to_additive] instance : Π [has_one α], has_one αᵒᵈ := id
```

Comment:
@urkud said that using `id` was a bad idea since it's one more thing to unfold
```suggestion
@[to_additive] instance [has_one α] : has_one αᵒᵈ := ‹has_one α›
```

Reply:
For future reference, here's a regex that works to turn one into the other: `: Π \[(.*)\],(.*)id`, `[h : $1] :$2h`